### ISSUE TITLE
Rust support for future?

### ISSUE THREAD

**Issue by eix128** [bytedeco/javacpp#359]
Well this project is for C++ i know but.
Rust is becoming more popular.
It would be good for support binding 
Java <=> Rust

Rust can import C++ libs.
Maybe it wont be hard to do this.

https://stackoverflow.com/questions/24105186/can-i-call-c-or-c-functions-from-rust-code

**Comment by saudet** [comment]
Yes, it's... *somewhere* on the roadmap... but not before issue #51 gets done. Not even sure when that one will happen because I'm pretty much the only guy working on these projects, but I'll assign @lloydmeta since he's now a Rust fan :)
